### PR TITLE
Colorful Rain Effect Added

### LINKED ISSUE #365: Adding colorful rain animation effect using HTML,CSS,Js
### Is your feature request related to a problem? Please describe.

A HTML site to show Colorful rain animation effect using CSS>

### Describe the solution you'd like.

A site that uses js timing function and displays amazing css effect.

### Describe alternatives you've considered.

No alternatives

### Add any other context or screenshots about the feature request here.

_No response_

### PR DESCRIPTION
**Pull request by rajprem4214**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

YES

### Description

A Colorful Rain Animation Effect Using HTML, CSS, JS.

Fixes #365 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

N/A

### CODE REVIEW

**Review by photon149 — CHANGES_REQUESTED** [review]
@rajprem4214 Add a README file with output SS.

**Comment by rajprem4214** [comment]
> @rajprem4214 Add a README file with output

Made required changes.

File: Web Development/Basic/Colorful Rain Effect/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
+}
```
Comment (Kushal997-das) [inline Web Development/Basic/Colorful Rain Effect/.vscode/settings.json:3]: Delete this file .

File: Web Development/Basic/Colorful Rain Effect/readme.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Built with :-
+
+- [x] HTML
+- [x] CSS
+- [x] JS
+
+## Screenshot
+
+![preview](https://user-images.githubusercontent.com/85401522/156910556-a22eecf1-4251-42ed-97ba-02c410b59456.png)
```
Comment (Kushal997-das) [inline Web Development/Basic/Colorful Rain Effect/readme.md:18]: Reduce the image size.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@rajprem4214 Do the changes !

File: Web Development/Basic/Colorful Rain Effect/readme.md (hunk 2 above)
Comment (rajprem4214) [reply Web Development/Basic/Colorful Rain Effect/readme.md]: > Reduce the image size.

done the same.

**Comment by rajprem4214** [comment]
> @rajprem4214 Do the changes !

made changes plz check

**Comment by rajprem4214** [comment]
> @rajprem4214 Do the changes !

@Kushal997-das  Made changes:  1. Reduced image size
                                                        2. Deleted Ignore File.

**Review by Kushal997-das — APPROVED** [review]
LGTM 👍 

@photon149 have a look !

**Review by photon149 — APPROVED** [review]
Approved from my end.

**Review by prakhar3062 — APPROVED** [review]

**Pull request merged**